src/harness/compiler.ts

```diff
@@ -0,0 +1,256 @@
+/// <reference path="./harness.ts" />

```

Comment:
we are removing all triple-slash references currently in favor of tsconfig.josn file list